process.sh

```diff
@@ -44,7 +44,7 @@ main() {
 
 ### Globals
 
-BASE=`dirname "$0"`
+BASE=`pwd`
```

Comment:
Why this change? The `dirname $0` idiom isn't perfect, but the idea is that it's relative to this script: Did you run into a problem?

Reply:
@thomaslchan : Could you explain this change?